SpectodaDummyConnector.js

```
@@ -153,20 +153,20 @@ export class SpectodaDummyConnector {
-    logging.verbose(`disconnect()`);
+    logging.verbose('disconnect()');
 
     return new Promise(async (resolve, reject) => {
       if (this.#connected) {
         await sleep(100); // disconnecting logic
         this.#connected = false;
-        this.#interfaceReference.emit("#disconnected");
+        this.#interfaceReference.emit('#disconnected');
       }
       resolve(); // always resolves even if there are internal errors
     });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)